src/React.js

```diff
@@ -65,5 +65,6 @@ assign(React, {
 });
 
 React.__SECRET_DOM_DO_NOT_USE_OR_YOU_WILL_BE_FIRED = ReactDOM;
+React.__SECRET_DOM_SERVER_DO_NOT_USE_OR_YOU_WILL_BE_FIRED = ReactDOMServer;
```

Comment:
Yup, this is fine as is. We are not planning on making these properties condoned in any public use. They are subject to breaking (and I might just change them in the next release to ensure this does break)